application/libraries/admin/quexmlpdf.php

```diff
@@ -3806,6 +3806,16 @@ protected function newPage($init = false)
             $this->SetHeaderMargin(0);
             $this->SetFooterMargin(0);
 
+            $oTemplate = Template::model()->getInstance();
```

Comment:
Could you please provide instructions on how to (manually) test this code? Something like

1. Go to x
2. Click on y
3. See z
4. ...

Thank you.

Reply:
Think it's hard to test pdf file ? Check if logo is shown in pdf file ?

Reply:
In your upload/themes/survey/{...}/files - folder must be the image defined in your application/config/config-defaults.php `pdflogofile` setting, then pdf queXML export (responses/sa/browse, second icon in list view) should display the image in the pdfs header

Reply:
Yes : but @olleharstedt talk for a way to add a [test](https://github.com/LimeSurvey/LimeSurvey/tree/develop/tests) for this. I don't think there are automatic way to add a test to check PDF content.

Reply:
Arg … sorry … **to (manually) test ** sorry both … please : delete my comments ;)